vaultpress.php

```diff
@@ -3046,12 +3046,9 @@ function register_via_jetpack( $already_purchased = false ) {
 
 if ( isset( $_GET['vaultpress'] ) && $_GET['vaultpress'] ) {
 	if ( !function_exists( 'wp_magic_quotes' ) ) {
-		// If already slashed, strip.
-		if ( get_magic_quotes_gpc() ) {
-			$_GET    = stripslashes_deep( $_GET    );
-			$_POST   = stripslashes_deep( $_POST   );
-			$_COOKIE = stripslashes_deep( $_COOKIE );
-		}
+		$_GET    = stripslashes_deep( $_GET    );
+		$_POST   = stripslashes_deep( $_POST   );
+		$_COOKIE = stripslashes_deep( $_COOKIE );
```

Comment:
PHP manual says that `get_magic_quotes_gpc` now always returns `false` (as the magic quotes  feature has been deprecated for quite a long an was finally removed in PHP8).

Hence, shouldn't we just get rid of these three lines, previously wrapped in an `if` statement?

Reply:
Makes sense, thanks for checking.